docs/release-notes/8.18.asciidoc

```diff
@@ -1,6 +1,27 @@
 [[release-notes-header-8.18.0]]
 == 8.18
 
+[discrete]
+[[release-notes-8.18.4]]
+=== 8.18.4
+
+[discrete]
+[[enhancements-8.18.4]]
+==== Enhancements
+* Improves logging of fatal exceptions in {elastic-endpoint}.
+
+[discrete]
+[[bug-fixes-8.18.4]]
+==== Fixes
+* Fixes differences between risk scoring preview and persisted risk scores ({kibana-pull}226456[#226456]).
+* Fixes domain validation in case observables ({kibana-pull}225901[#225901]).
+* Updates a placeholder and validation message in the **Related Integrations** section of the rule upgrade flyout ({kibana-pull}225775[#225775]).
+* Excludes {ml} rules from installation and upgrade checks for users with Basic or Essentials licenses ({kibana-pull}224676[#224676]).
+* Allows using days as a time unit in rule schedules, fixing an issue where durations normalized to days were incorrectly displayed as 0 seconds ({kibana-pull}224083[#224083]).
+* Strips `originId` from connectors before rule import to ensure correct ID regeneration and prevent errors when migrating connector references on rules ({kibana-pull}223454[#223454]).
+* Refactors Timeline styling for improved consistency with design updates ({kibana-pull}222438[#222438]).
+* Fixes a bug where the **Rules**, **Alerts**, and **Fleet** pages would stall in air-gapped environments by ensuring API requests are sent even when offline ({kibana-pull}220510[#220510]).
```

Comment:
* {elastic-defend} users can now control the maximum file size for malware protection via the `advanced.malware.max_file_size_bytes` advanced policy option.
* Fixes an issue in {elastic-defend} that may result in bugchecks (BSODs) on Windows systems with a very high volume of network connections. This issue has only been observed on Windows Server.